app/server/controllers/UserController.js

```diff
@@ -663,11 +663,30 @@ UserController.updateRecordsWithMissingFields = function(callback) {
     .exec(callback);
 }
 
+/**
+ * Returns if everyone in the team has checked in
+ * @param  {Function} callback [description]
+ */
+UserController.teamIsCompletelyCheckedIn = function(id, callback) {
+  this.getTeammates(id, function(err, teammates) {
+    if(err) return callback(err, teammates);
+
+    let everyoneIsCheckedIn = true;
+    teammates.forEach(function(member) {
```

Comment:
This needst to be changed to count the number of checked in and compare it to `teamSizeAccepted` instead of using a boolean